registration/models.py

```diff
@@ -628,6 +629,9 @@ class SupervisedRegistrationProfile(RegistrationProfile):
     # in admin approval
     objects = SupervisedRegistrationManager()
 
+    class Meta:
```

Comment:
Is this necessary? `SupervisedRegistrationProfile` inherits from `RegistrationProfile`